Makefile

```diff
@@ -36,6 +36,14 @@ bench:
 	$(info INFO: Starting build $@)
 	go test -bench=.
 
+alpha-test-%: release/goss-%
+	$(info INFO: Starting build $@)
+	./integration-tests/run-tests-alpha.sh $*
+
+test-serve-%: release/goss-%
```

Comment:
make test should execute this as part of it's testing, so that it's part of CI.

I was able to test it locally with `make test-serve-linux-amd64` and it looks great.

Reply:
It's wired via `ci/build.sh` - or have I made a mistake?